deployment-templates/compute-engine/server/forseti-instance-server.py

```diff
@@ -235,11 +235,16 @@ def GenerateConfig(context):
 gsutil cp gs://{scanner_bucket}/configs/forseti_conf_server.yaml {forseti_server_conf}
 gsutil cp -r gs://{scanner_bucket}/rules {forseti_home}/
 
+# Download the Newest Config Validator constraints from GCS
+rm -rf /home/ubuntu/config_validator_constraints
+gsutil cp -r gs://{scanner_bucket}/config_validator_constraints /home/ubuntu/
+
 # Start Forseti service depends on vars defined above.
 bash ./install/gcp/scripts/initialize_forseti_services.sh
 
 echo "Starting services."
 systemctl start cloudsqlproxy
+systemctl start config-validator
```

Comment:
It will if there is a policy-library folder under the forseti server gcs bucket. This is a manual step the user needs to take. The validator instance will be reset every time the config validator scanner is finished running so the impact should be minimal when the config validator scanner is not running

Reply:
Got it.